From 1d048a8f2caec10eb0c9647f0554a60be619602d Mon Sep 17 00:00:00 2001
From: Andreas Boll <andreas.boll.dev@gmail.com>
Date: Mon, 5 May 2014 23:28:23 +0200
Subject: [PATCH libdrm] libdrm: Fix drm.h include in qxl drm header file

Use "drm.h" instead of "drm/drm.h" as used in the other header files.
Fixes xserver-xorg-video-qxl build with KMS support on Debian, where this
file is installed in /usr/include/libdrm.

Fixes Debian bug #746807

Reported-by: Bastian Blank <waldi@debian.org>
Signed-off-by: Andreas Boll <andreas.boll.dev@gmail.com>
---
 include/drm/qxl_drm.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/drm/qxl_drm.h b/include/drm/qxl_drm.h
index 5d7de70..1e331a8 100644
--- a/include/drm/qxl_drm.h
+++ b/include/drm/qxl_drm.h
@@ -25,7 +25,7 @@
 #define QXL_DRM_H
 
 #include <stddef.h>
-#include "drm/drm.h"
+#include "drm.h"
 
 /* Please note that modifications to all structs defined here are
  * subject to backwards-compatibility constraints.
-- 
1.9.2

